<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>478</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>12</y>
      <width>910</width>
      <height>512</height>
     </rect>
    </property>
    <property name="viewMode">
     <enum>QMdiArea::TabbedView</enum>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrintPreview"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Preferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuToolbars">
     <property name="title">
      <string>Toolbars</string>
     </property>
     <addaction name="actionLayers"/>
     <addaction name="actionBlocks"/>
    </widget>
    <addaction name="actionGrid"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionPrevious_View"/>
    <addaction name="actionWindows_Zoon"/>
    <addaction name="actionZoom_Panning"/>
    <addaction name="separator"/>
    <addaction name="actionStatus_Bar"/>
    <addaction name="menuToolbars"/>
   </widget>
   <widget class="QMenu" name="menuSelect">
    <property name="title">
     <string>Select</string>
    </property>
    <addaction name="actionSelect_All"/>
    <addaction name="actionDeselect_All"/>
    <addaction name="actionSelect_Entity"/>
    <addaction name="action_De_elect_Contour"/>
    <addaction name="actionSelect_Window"/>
    <addaction name="actionDeselect_Window"/>
    <addaction name="actionInverted_Selection"/>
    <addaction name="actionSelect_Intersected_Entities"/>
    <addaction name="actionDeselect_Intersected_Entities"/>
    <addaction name="action_De_Select_Layer"/>
   </widget>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Draw</string>
    </property>
    <addaction name="actionPoints"/>
    <addaction name="actionLine"/>
    <addaction name="actionArc"/>
    <addaction name="actionCircle"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionSpline"/>
    <addaction name="actionPolyline"/>
    <addaction name="actionMText"/>
    <addaction name="actionText"/>
    <addaction name="actionHatch"/>
    <addaction name="actionInsert_Image"/>
   </widget>
   <widget class="QMenu" name="menuModify">
    <property name="title">
     <string>Modify</string>
    </property>
    <addaction name="actionMove_Copy"/>
    <addaction name="actionRotate"/>
    <addaction name="actionScale"/>
    <addaction name="actionMirror"/>
    <addaction name="actionMove_and_Rotate"/>
    <addaction name="actionRotate_Two"/>
    <addaction name="actionRevert_Direction"/>
    <addaction name="actionTrim"/>
    <addaction name="actionTrim_Two"/>
    <addaction name="actionLengthen"/>
    <addaction name="actionBevel"/>
    <addaction name="actionRound"/>
    <addaction name="actionDivid"/>
    <addaction name="actionStretch"/>
    <addaction name="actionProperties"/>
    <addaction name="actionAttributes"/>
    <addaction name="actionDelete"/>
    <addaction name="actionDelete_Selected"/>
    <addaction name="actionExplode"/>
    <addaction name="actionAlign"/>
   </widget>
   <widget class="QMenu" name="menuDimension">
    <property name="title">
     <string>Dimension</string>
    </property>
    <addaction name="actionAligned"/>
    <addaction name="actionLinear"/>
    <addaction name="actionHorizontal"/>
    <addaction name="actionVertical"/>
    <addaction name="actionRadial"/>
    <addaction name="actionDiametric"/>
    <addaction name="actionAngular"/>
    <addaction name="actionLeader"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSelect"/>
   <addaction name="menuDraw"/>
   <addaction name="menuModify"/>
   <addaction name="menuDimension"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPoints"/>
   <addaction name="actionLine"/>
   <addaction name="actionCircle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionMText"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Print </string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Print Preview</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Cut </string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionEdit_Preferences">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit Preferences</string>
   </property>
  </action>
  <action name="actionGrid">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Grid</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionPrevious_View">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Previous View</string>
   </property>
  </action>
  <action name="actionWindows_Zoon">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Windows Zoom</string>
   </property>
  </action>
  <action name="actionZoom_Panning">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Zoom Panning</string>
   </property>
  </action>
  <action name="actionStatus_Bar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Status Bar</string>
   </property>
  </action>
  <action name="actionLayers">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Layers</string>
   </property>
  </action>
  <action name="actionBlocks">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Blocks</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionDeselect_All">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Deselect All</string>
   </property>
  </action>
  <action name="actionSelect_Entity">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select Entity</string>
   </property>
  </action>
  <action name="action_De_elect_Contour">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>(De)-elect Contour</string>
   </property>
  </action>
  <action name="actionSelect_Window">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select Window</string>
   </property>
  </action>
  <action name="actionDeselect_Window">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Deselect Window</string>
   </property>
  </action>
  <action name="actionInverted_Selection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Inverted Selection</string>
   </property>
  </action>
  <action name="actionSelect_Intersected_Entities">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Select Intersected Entities</string>
   </property>
  </action>
  <action name="actionDeselect_Intersected_Entities">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Deselect Intersected Entities</string>
   </property>
  </action>
  <action name="action_De_Select_Layer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>(De)-Select Layer</string>
   </property>
  </action>
  <action name="actionPoints">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/images/points.png</normaloff>:/icons/images/points.png</iconset>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/images/line.png</normaloff>:/icons/images/line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
  </action>
  <action name="actionArc">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Arc</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/images/circles.png</normaloff>:/icons/images/circles.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/images/ellipse.png</normaloff>:/icons/images/ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionSpline">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Spline</string>
   </property>
  </action>
  <action name="actionPolyline">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Polyline</string>
   </property>
  </action>
  <action name="actionMText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/icons/images/menutext.png</normaloff>:/icons/images/menutext.png</iconset>
   </property>
   <property name="text">
    <string>MText</string>
   </property>
  </action>
  <action name="actionText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionHatch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Hatch</string>
   </property>
  </action>
  <action name="actionInsert_Image">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Insert Image</string>
   </property>
  </action>
  <action name="actionMove_Copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Move/Copy</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </action>
  <action name="actionScale">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </action>
  <action name="actionMirror">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
  </action>
  <action name="actionMove_and_Rotate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Move and Rotate</string>
   </property>
  </action>
  <action name="actionRotate_Two">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate Two</string>
   </property>
  </action>
  <action name="actionRevert_Direction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Revert Direction</string>
   </property>
  </action>
  <action name="actionTrim">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Trim</string>
   </property>
  </action>
  <action name="actionTrim_Two">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Trim Two</string>
   </property>
  </action>
  <action name="actionLengthen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Lengthen</string>
   </property>
  </action>
  <action name="actionBevel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Bevel</string>
   </property>
  </action>
  <action name="actionRound">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Round</string>
   </property>
  </action>
  <action name="actionDivid">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Divide</string>
   </property>
  </action>
  <action name="actionStretch">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stretch</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionAttributes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Attributes</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionDelete_Selected">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete Selected</string>
   </property>
  </action>
  <action name="actionExplode">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Explode</string>
   </property>
  </action>
  <action name="actionAlign">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Align</string>
   </property>
  </action>
  <action name="actionAligned">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Aligned</string>
   </property>
  </action>
  <action name="actionLinear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Linear</string>
   </property>
  </action>
  <action name="actionHorizontal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Horizontal</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Vertical</string>
   </property>
  </action>
  <action name="actionRadial">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Radial</string>
   </property>
  </action>
  <action name="actionDiametric">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Diametric</string>
   </property>
  </action>
  <action name="actionAngular">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Angular</string>
   </property>
  </action>
  <action name="actionLeader">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Leader</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manual</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
